<launch>

    <arg name="urdf_path" default="walker.urdf" />

    <node type="solver_server_node" name="solver_server_node" pkg="thewalkingdead" output="screen">
        <param name="walker_urdf_path" value="$(arg urdf_path)" />
    </node>

</launch>